mappings/net/minecraft/world/biome/layer/AddColdBiomeClimatesLayer.mapping

```diff
@@ -0,0 +1 @@
+CLASS clu net/minecraft/world/biome/layer/AddColdBiomeClimatesLayer
```

Comment:
Here you're calling them "biome climates", but in `AddClimateLayers` you call them just "climates" I think the `Biome` should be dropped here for consistency. Also, it's adding just one climate (the cold climate), so it should be `AddColdClimateLayer` (or equivalently, `AddColdBiomesLayer` since one climate is many biomes).

Reply:
ok

Reply:
Ignore the second part of my comment, it's adding two climates